examples/c/src/sum.c

```diff
@@ -0,0 +1,7 @@
+#include "sum.h"
+
+int sum(int n1, int n2)
+{
```

Comment:
The style of this function is different from the style used in the main function (i.e. the main function has the curly brace on the same line as the function, but this one is on the second line). Can you pick a single style and use it consistently.